app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/dto/Furniture.kt

```diff
@@ -7,9 +7,9 @@ data class Furniture(
     @SerializedName("video_link")var videoLink: String,
     @SerializedName("article_number")var articleNumber: String,
     @SerializedName("furniture_name") var furnitureName: String,
-    @SerializedName("instructions_manuel")var instructionsManuel: String
+    @SerializedName("instructions_manual")var instructionsManuel: String
```

Comment:
Spelling error.

Reply:
Yes... I noticed this one earlier.  Thanks for fixing.  :)